README.md

```diff
@@ -0,0 +1,6 @@
+# [Stackdriver Monitoring Agent](https://cloud.google.com/monitoring/agent/)
```

Comment:
Technically speaking, our forked `Collectd` is not equal to `Stackdriver Monitoring Agent`. The agent uses this fork, but it involves additional packaging and configurations.

How about?
# Collectd fork for [Stackdriver Monitoring Agent](https://cloud.google.com/monitoring/agent/)